theano/sandbox/cuda/opt_util.py

```diff
@@ -100,9 +101,7 @@ def opt(node):
                     # other cases are too complex for now
                     return None
                 if W.broadcastable != targ.inputs[out_in].broadcastable:
-                    # Would need to explicitly tile the output to fill
-                    # the full shape here.  Disable for now.
-                    return None
+                    W = broadcast_like(W, targ.inputs[out_in], node.fgraph)
```

Comment:
I disagree, but I don't want to argue too much since there are other parts of this PR that fix problems, so I'll just remove that part.